sass/components/_popover.scss

```diff
@@ -121,36 +121,37 @@
         }
     }
 
+    /** TODO: are these rules needed? They are hidden in the rules above **/
```

Comment:
I think this could be better organized activating some of the linter rules that are not activated. Any thoughts on that?